docs/plugins.md

```diff
@@ -169,48 +154,6 @@ Your `.babelrc`:
 }
 ```
 
-## Plugin/Preset Paths
-
-If the plugin is on npm, you can pass in the name of the plugin and babel will check that it's installed in `node_modules`
-
-```json
-{
-  "plugins": ["babel-plugin-myPlugin"]
-}
-```
-
-You can also specify an relative/absolute path to your plugin.
-
-```json
-{
-  "plugins": ["./node_modules/asdf/plugin"]
-}
-```
-
-### Plugin Shorthand
```

Comment:
this is confusing and we recommend using the full path now anyway so can remove? https://github.com/babel/website/issues/2244

Have a link to options.md#name-normalization (it's sorta hidden) but it's there